### DISCUSSION TITLE
✌️

### DISCUSSION THREAD

**Discussion by AressAntonio** [Launch-X-Latam/MisionFrontEnd#475] [Team Posts]
Hola , a todos ✌️

**Comment by AlmaLSt** [comment]
Holis 😎

**Comment by FerIqm1993** [comment]
Hola

**Comment by Arego1994** [comment]
Holi

**Comment by manutoledo21** [comment]
Che, no es necesario que manden mensajes acá, le llegan correos a todos por cada mensaje que escriben acá. No molesten por favor, de aquí a que veamos back, falta mucho. No lo tomen a mal, pero ya me llegaron 3 mails con mensajes de acá... Saludos!

**Comment by AoGnyan** [comment]
> Che, no es necesario que manden mensajes acá, le llegan correos a todos por cada mensaje que escriben acá. No molesten por favor, de aquí a que veamos back, falta mucho. No lo tomen a mal, pero ya me llegaron 3 mails con mensajes de acá... Saludos!

a la izquierda abajo de donde dice mision backend puedes cambiar los settings para las notificaciones, donde dice watch justo
![image](https://user-images.githubusercontent.com/88294155/195037304-7ccb8da7-b117-47bf-8cea-00ea9307f746.png)
![image](https://user-images.githubusercontent.com/88294155/195037342-e66e0ebf-059e-4ebb-b924-151e89a34ff8.png)
ahi justo

**Comment by jjmillar** [comment]
Llegamos!
